accepted/automatic-redirects-by-default/automatic-redirects-by-default.md

```diff
@@ -85,6 +85,37 @@ being on by default. Otherwise his application would have crashed with a
 
 ### Goals
 
+* **Turned off if the project isn't targeting .NET Framework**. The .NET
+  Framework is the only runtime that needs binding redirects. While computing
+  conflicts for other runtimes doesn't do any harm but it does impact
+  performance as the MSBuild has to walk more assemblies.
+
+* **Turned off in class libraries**. Binding redirects are generally not useful
+  for class libraries as they are for the application. Again, computing them
+  doesn't do harm but it does impact performance as the MSBuild has to walk more
+  assemblies.
+
+* **Turned off in web projects**. Web projects cannot use an `app.config` file
+  as they use `web.config`, which is usually taken from the source directory. At
+  best, dropping the extra file is just noise but at worse it might confuse the
+  build or the runtime.
+
+* **Automatically turned on when targeting .NET Framework 4.6.1 or higher and a
+  .NET Standard 2.0 library is consumed**. This addresses the primary issues
```

Comment:
The extensions code injects facades/libs when referencing .NET Standard 1.5 or greater IIRC. I presume that we should be enabling it for the same cases.

Reply:
Excellent point. Clarified.